core/src/test/java/apoc/trigger/TriggerNewProceduresTest.java

```diff
@@ -0,0 +1,583 @@
+    public void testDeleteRelationships() {
+        db.executeTransactionally("CREATE (a:A {name: \"A\"})-[:R1]->(z:Z {name: \"Z\"}), (a)-[:R2]->(z)");
+        String name = "trigger-after-async-3";
+        final String query = "UNWIND $deletedRelationships AS r\n" +
+                "MERGE (a:AA{name: \"AA\"})\n" +
+                "SET a.triggerAfter = size($deletedRelationships) = 1, a.deleted = type(r)";
+        sysDb.executeTransactionally("CALL apoc.trigger.install('neo4j', $name, $query, {phase: 'after'})",
+                Map.of("name", name, "query", query));
+        awaitTriggerDiscovered(db, name, query);
+
+
+        db.executeTransactionally("MATCH (a:A {name: \"A\"})-[r:R2]->(z:Z {name: \"Z\"})\n" +
+                "DELETE r");
+
+        assertEventually(() ->
+                        db.executeTransactionally("MATCH (a:AA) RETURN a", Map.of(),
+                                result -> {
+                                    final Node r = result.<Node>columnAs("a").next();
+                                    return (boolean) r.getProperty("triggerAfter", false)
+                                            && r.getProperty("deleted", "").equals("R2");
+                                })
+                , (value) -> value, 30L, TimeUnit.SECONDS);
+    }
+    
+    //
+    // new test cases
+    //
+
+    @Test
+    public void testTriggerShow() {
+        String name = "test-show1";
+        String name2 = "test-show2";
+        String query = "MATCH (c:TestShow) SET c.count = 1";
+        
+        testCall(sysDb, "CALL apoc.trigger.install('neo4j', $name, $query,{}) YIELD name",
+                map("query", query, "name", name),
+                r -> assertEquals(name, r.get("name")));
+        
+        testCall(sysDb, "CALL apoc.trigger.install('neo4j', $name, $query,{}) YIELD name",
+                map("query", query, "name", name2),
+                r -> assertEquals(name2, r.get("name")));
+        
+        // not updated
+        testResult(sysDb, "CALL apoc.trigger.show('neo4j') " +
+                        "YIELD name, query RETURN * ORDER BY name",
+                map("query", query, "name", name),
+                res -> {
+                    Map<String, Object> row = res.next();
+                    assertEquals(name, row.get("name"));
+                    assertEquals(query, row.get("query"));
+                    row = res.next();
+                    assertEquals(name2, row.get("name"));
+                    assertEquals(query, row.get("query"));
+                    assertFalse(res.hasNext());
+                });
+    }
+    
+    @Test
+    public void testInstallTriggerInUserDb() {
+        try {
+            testCall(db, "CALL apoc.trigger.install('neo4j', 'userDb', 'RETURN 1',{})",
+                    r -> fail("Should fail because of user db execution"));
+        } catch (QueryExecutionException e) {
+            assertTrue(e.getMessage().contains(TRIGGER_NOT_ROUTED_ERROR));
+        }
+    }
```

Comment:
Could we add a test that installs a trigger in a user database which doesn't exist? The test should fail